### PR TITLE
16 wallet integration

### PR DESCRIPTION
**Pull request by yoon-bbox**
Integrated Phantom wallet with other wallet options including Ledger, Solfare and Sollet using [Wallet Adapter](https://github.com/solana-labs/wallet-adapter) and customized the theme to our ecoverse theme.

Screen Shot 2023-01-16 at 4 00 57 PM

Screen Shot 2023-01-16 at 3 59 56 PM

Screen Shot 2023-01-16 at 4 02 38 PM

### CODE REVIEW

File: components/WalletConnectButton.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { WalletMultiButton } from '@solana/wallet-adapter-react-ui';
+
+const WalletConnectButton = () => {
+  return (
+    <div>
+      <div className="w-full mx-1 text-primary hover:text-secondary">
```
Comment (jekiquin-bb) [inline components/WalletConnectButton.tsx]: Styles object. looool. Sorry.

Also, you have an outer div before this. Any reason for that?

File: context/wallet.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+} from '@solana/wallet-adapter-wallets';
+import { clusterApiUrl } from '@solana/web3.js';
+import { ChildrenProps as Props } from '@utils/interface/global-interface';
+import React, { FC, ReactNode, useCallback, useMemo } from 'react';
+
+const WalletContextProvider: FC<{ children: ReactNode }> = ({
+  children,
+}: Props) => {
+  // Todo: change the network based on the node env
+  const network = WalletAdapterNetwork.Devnet;
```
Comment (jekiquin-bb) [inline context/wallet.tsx]: Based from the todo that you have above, we can probably add a wallet config under the `config` folder where you can include this. Not needed at the moment, but just a quick note. Can be defaulted to devnet if nothing is written on env file. 

ps. not sure how to add a normal comment on the code so I'm just adding this as a review comment. lol. Just resolve this if you read it :D

**Review by jekiquin-bb — CHANGES_REQUESTED** [review]
Just minor changes and a comment that you can ignore. lol

Everything else looks good! Nice one Yoon!

File: components/WalletConnectButton.tsx (hunk 1 above)
Comment (yoon-bbox) [reply components/WalletConnectButton.tsx]: ahhh forgot again sorry i'll make sure to double check next time!

**Review by jekiquin-bb — APPROVED** [review]

**Pull request merged**